examples/deploy/basic/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Maybe this should be changed to `!/usr/bin/env sh`. It is the standard way to make sure this deploy will work on all kinds of systems. /usr/bin/env can interpret your $PATH, which makes scripts more portable. https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my

Reply:
The question ends up being, do more systems have `/bin/sh` or `/usr/bin/env`?? /troll

Reply:
Also I am pretty sure thats almost exclusively for things like python, perl or ruby which may need to run from different versions, pretty sure no one really does that for sh or even bash.